src/util/tubemap.js

```diff
@@ -682,10 +684,32 @@ function placeReads() {
           previousValidY = reads[idx].path[lastIndex].y;
           lastIndex = lastIndex - 1;
         }
+
+        // sometimes, elements without nodes are between 2 segments going to the same node, from the same direction
+        // this means we're looping back to the same node, which constitutes a different sorting priority
+        
+        let previousNodeOrder = reads[idx].path[pathIdx - 1]?.order;
+        let nextNodeOrder = reads[idx].path[pathIdx + 1]?.order;
+
+        let previousNodeIsForward = reads[idx].path[pathIdx - 1]?.isForward;
+        let nextNodeIsForward = reads[idx].path[pathIdx + 1]?.isForward
+
+        // if the previous segment and the next segment is going to the same node
+        let sameNode = previousNodeOrder !== null && nextNodeOrder !== null && previousNodeOrder === nextNodeOrder;
+        // if the previous semgne tnad the next segment is going in the same direction
+        let sameDirection = previousNodeIsForward !== null && nextNodeIsForward !== null && previousNodeIsForward === nextNodeIsForward
+
+        let betweenCycle = false;
+        // if the previous segment and the next segment is going to the same node
+        if (sameNode && sameDirection) {
+          betweenCycle = true;
+        }
```

Comment:
I think we might want to use this reverse sort order logic in more cases than *just* the one where we visit the same node in the same orientation before and after the no-node section.

I think we want to have the same behavior even if that one node that we loop around is instead divided up into two or more nodes that we loop around as a group. Like it the read visits nodes 1, 2, 1, 2, 1, 2. I'm not sure of the best way to detect that we are visiting no node because we need to get back to an earlier X to complete a cycle, though.